pkg/distributor/distributor_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"google.golang.org/grpc/status"
```

Comment:
This import needs to be separate from the standard library imports. We run rather strict linter settings for the project and this is triggering the CI failure.

Reply:
ahh, good to know, thanks!

Reply:
just a bit more detail about the linting: https://cortexmetrics.io/docs/contributing/#formatting